app/Commands/UploadTrackCommand.php

```diff
@@ -48,12 +62,16 @@ class UploadTrackCommand extends CommandBase
      * @param bool $allowShortTrack allow tracks shorter than 30 seconds
      * @param string|null $customTrackSource value to set in the track's "source" field; if left blank, "direct_upload" is used
      * @param bool $autoPublishByDefault
+     * @param int $version
+     * @param Track $track | null
      */
     public function __construct(
         bool $allowLossy = false,
         bool $allowShortTrack = false,
         string $customTrackSource = null,
-        bool $autoPublishByDefault = false
+        bool $autoPublishByDefault = false,
+        int $version = 1,
```

Comment:
Why is the version number part of this command's interface? It should be an implementation detail given that audio versions are immutable.